src/tests/system/tests/test_minimal.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.ticket(bz=1654018)
+@pytest.mark.topology(KnownTopology.Client)
+def test_minimal__local_user_authentication(client: Client):
```

Comment:
Minimal profile is no longer available in master, (RHEL-10) please,  replace it with local.

Keep minimal for 1.2.x branch that should be used for RHEL-8,9.

Reply:
I'll create another pull request for the 1.2.x branch after this one, these tests will apply to 1.2.x , so I think a backport will be fine?